src/query/api/v1/httpd/handler.go

```diff
@@ -62,7 +59,6 @@ var (
 // Handler represents an HTTP handler.
 type Handler struct {
 	Router        *mux.Router
-	CLFLogger     *log.Logger
```

Comment:
Why are we removing the logger here?

Reply:
we weren't using it so I figured better to remove